src/common/serviceUtils.js

```diff
@@ -118,3 +124,21 @@ export function handleRequestError(error) {
   logAPIErrorResponse(error);
   throw error;
 }
+
+export function isUnknownError(error) {
```

Comment:
Helps us know which errors are unknown in the sagas, so we can log them as errors (and not just log them to info above)

Reply:
I've got a slightly different design in mind, but I think we can land this and loop back.  Happy to discuss my thoughts as well, but quicker in person.